### PR TITLE
Deprecation cleanup

### PR DESCRIPTION
**Pull request by markt-asf**

### CODE REVIEW

**Comment by markt-asf** [comment]
That should be it for removal of deprecated code / features.

File: api/pom.xml [hunk 1]
```diff
@@ -93,11 +93,26 @@
...
+            <version>6.0.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
 
+    <repositories>
+      <repository>
+        <name>Jakarta EE Snapshots</name>
+        <id>snapshots-repo</id>
+        <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots/</url>
```
Comment (joakime) [inline api/pom.xml:105]: Isn't this part of the parent pom?

File: api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2021 Oracle and/or its affiliates and others.
+ * Copyright (c) 1997, 2022 Oracle and/or its affiliates and others.
```
Comment (joakime) [inline api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java:2]: Why did you update this to 2022, but not the others?

**Review by joakime — COMMENTED** [review]
Some questions really, otherwise it looks good.

File: api/pom.xml (hunk 1 above)
Comment (markt-asf) [reply api/pom.xml:105]: Yes it is. My mistake for thinking the "effective POM" view in Eclipse would show me all the profiles. -Psnapshots does the trick. I'll drop this commit from the PR.

File: api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java (hunk 2 above)
Comment (markt-asf) [reply api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java:2]: Forgetfulness. I need to get back into the habit of updating these. I'll get the commits updated.

**Review by joakime — APPROVED** [review]
:+1:

**Pull request merged**